site/content/en/docs/manual/advanced/formats/format-yolov8.md

```diff
@@ -0,0 +1,117 @@
+---
+title: 'YOLOv8'
+linkTitle: 'YOLOv8'
+weight: 7
+description: 'How to export and import data in YOLOv8 formats'
+---
+
+YOLOv8 is a format family which consists of four formats:
+- [Detection](https://docs.ultralytics.com/datasets/detect/)
+- [Oriented bounding Box](https://docs.ultralytics.com/datasets/obb/)
+- [Segmentation](https://docs.ultralytics.com/datasets/segment/)
+- [Pose](https://docs.ultralytics.com/datasets/pose/)
+
+## YOLOv8 export
+
+For export of images:
+
+- Supported annotations
+  - Detection: Bounding Boxes
+  - Oriented bounding box: Oriented Bounding Boxes
+  - Segmentation: Polygons
+  - Pose: Skeletons
+- Attributes: Not supported.
+- Tracks: Not supported.
+
+The downloaded file is a .zip archive with the following structure:
+
+```bash
+archive.zip/
+   ├── data.yaml  # configuration file
+   ├── train.txt  # list of train subset image paths
+   │
+   ├── images/
+   │   ├── train/  # directory with images for train subset
+   │   │    ├── image1.jpg
+   │   │    ├── image2.jpg
+   │   │    ├── image3.jpg
+   │   │    └── ...
+   ├── labels/
+   │   ├── train/  # directory with annotations for train subset
+   │   │    ├── image1.txt
+   │   │    ├── image2.txt
+   │   │    ├── image3.txt
+   │   │    └── ...
+
+# train.txt:
+images/<subset>/image1.jpg
+images/<subset>/image2.jpg
+...
+
+# data.yaml:
+path:  ./ # dataset root dir
+train: train.txt  # train images (relative to 'path')
+
+# YOLOv8 Pose specific field
+# First number is the number of points in each skeleton
+# Second number defines the format of point info in annotation txt files
+kpt_shape: [17, 3]
+
+# Classes
+names:
+  0: person
+  1: bicycle
+  2: car
+  # ...
+
+# <image_name>.txt:
+# content depends on format
+
+# YOLOv8 Detection:
+# label_id - id from names field of data.yaml
+# cx, cy - relative coordinates of the bbox center
+# rw, rh - relative size of the bbox
+# label_id cx cy rw rh
+1 0.3 0.8 0.1 0.3
+2 0.7 0.2 0.3 0.1
+
+# YOLOv8 Oriented Bounding Boxes:
+# xn, yn - relative coordinates of the n-th point
+# label_id x1 y1 x2 y2 x3 y3 x4 y4
+1 0.3 0.8 0.1 0.3 0.4 0.5 0.7 0.5
+2 0.7 0.2 0.3 0.1 0.4 0.5 0.5 0.6
+
+# YOLOv8 Segmentation:
+# xn, yn - relative coordinates of the n-th point
+# label_id x1 y1 x2 y2 x3 y3 ...
+1 0.3 0.8 0.1 0.3 0.4 0.5
+2 0.7 0.2 0.3 0.1 0.4 0.5 0.5 0.6 0.7 0.5
+
+# YOLOv8 Pose:
```

Comment:
I also would recommend to add here a line about exporting multiple different skeletons. Need to explain how they are going to be exported and imported. I'm sure we will have huge amount of questions about that. It is also good to add a link on the original issue on the ultralytics github.

Reply:
Added a note about what happens when there are several skeletons